src/util/LimitedCollection.js

```diff
@@ -100,11 +100,11 @@ class LimitedCollection extends Collection {
   /**
    * Options for generating a filter function based on lifetime
    * @typedef {Object} LifetimeFilterOptions
-   * @property {number} [lifetime=14400] How long an entry should stay in the collection before it is considered
-   * sweepable.
-   * @property {Function} [getComparisonTimestamp=`e => e.createdTimestamp`] A function that takes an entry, key,
+   * @property {number} [lifetime=14400] How long, in seconds, an entry should stay in the collection
+   * before it is considered sweepable.
+   * @property {Function} [getComparisonTimestamp='e => e?.createdTimestamp'] A function that takes an entry, key,
```

Comment:
The `'` render as-is on the docs, just omitting them works fine for me:
```suggestion
   * @property {Function} [getComparisonTimestamp=e => e?.createdTimestamp] A function that takes an entry, key,
```

Reply:
Could've sworn I had done this one already bruh